components/global/product_switcher_menu/product_switcher_menu.tsx

```diff
@@ -40,6 +41,12 @@ type Props = {
     intl: IntlShape;
     firstAdminVisitMarketplaceStatus: boolean;
     onClick?: (event: React.MouseEvent<HTMLElement>) => void;
+    prevTrialLicense: ClientLicense;
+    actions: {
+        getPrevTrialLicense: () => void;
+        getLicenseConfig: () => void;
+        getStandardAnalytics: () => void;
+    };
```

Comment:
Could we move these into the `Menu.StartTrial` component so that it's not repeated in all places it will be added?

Reply:
More of a chicken and egg problem. The Menu.StartTrial itself shows or hides from the menu based on state set by those calls